src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx

```diff
@@ -53,6 +53,14 @@ class TimeRangeSelector extends React.Component {
      * Callback when "Update" button is clicked
      */
     onUpdate: PropTypes.func,
+    /**
+     * Align contents
+     */
+    align: PropTypes.oneOf(['right', 'left']),
```

Comment:
What is this used for? I think `HeaderItem` is a `Flex` component and `left` and `right` aren't valid flexbox values for `align-items`.